src/main/java/com/terraformersmc/modmenu/gui/widget/entries/ParentEntry.java

```diff
@@ -151,9 +150,9 @@ private void toggleChildren() {
 	}
 
 	@Override
-	public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
+	public boolean keyPressed(KeyInput input) {
 		String modId = getMod().getId();
-		if (keyCode == GLFW.GLFW_KEY_ENTER || keyCode == GLFW.GLFW_KEY_SPACE) {
+		if (input.key() == GLFW.GLFW_KEY_ENTER || input.key() == GLFW.GLFW_KEY_SPACE) {
```

Comment:
```suggestion
		if (input.isEnterOrSpace()) {
```

This change is technically a behavioral change by allowing keypad enter to be used. Did that behavior change for vanilla user interface elements in Minecraft 1.21.9?